data/helper/data_dictionary.csv

```diff
@@ -57,29 +57,83 @@ Membership,created_by,acronym of project member creating this line,,2018-01-30,O
-Text,commentary,Space for background information on the text. If necessary; for example reference to translation of a poem into English; on when a text was written…,,2017-07-04,LH,2017-07-04,LH
-Text,note,This column is a metacategory and only serves as field of communication between the project members with regards to this very entry,,2017-10-14,OS,2017-10-14,DP
-Text,created,date of creation of line,,2017-07-04,LH,2017-07-04,LH
-Text,created_by,acronym of project member creating this line,,2017-07-04,LH,2017-07-04,LH
-Text,last_modified,date of last modification,,2017-07-04,LH,2017-07-04,LH
-Text,last_modified_by,acronym of project member last modifying this line,,2017-07-04,LH,2017-07-04,LH
+ArtWork,artwork_id,AW and a five-digit number (leading zeros),,2018-02-09,OS,2018-02-09,OS
+ArtWork,title,Title of an artwork; both in the original language and Chinese. If the Original language is not either English or French or German; add English.,,2018-02-09,OS,2018-02-09,OS
+ArtWork,subtitle,Subtitle of an artwork; both in the original language and Chinese. If the Original language is not either English or French or German; add English.,,2018-02-09,OS,2018-02-09,OS
+ArtWork,title_lang,Language code of this title ('en' or 'zh' or 'ru' etc.). ,,2018-02-09,OS,2018-02-09,OS
+ArtWork,person_id,key: Person.person_id,,2018-02-09,OS,2018-02-09,OS
+ArtWork,art_form_id,key: art_form.genre_id,,2018-02-09,OS,2018-02-09,OS
+ArtWork,first_performance_date,date of first performance or showing,,2018-02-09,OS,2018-02-09,OS
+ArtWork,first_performance_place_id,place of first performance or showing; key: Place.place_id ,,2018-02-09,OS,2018-02-09,OS
+ArtWork,neibu,yes/no whether this was originally a neibu publication. Add 'NULL' for every artwork before 1949 and after 1980.,,2018-02-09,OS,2018-02-09,OS
+ArtWork,commentary,Space for background information on the trtworkeaxt. If necessary; for example a reference to a world premiere of a movie,,2018-02-09,OS,2018-02-09,OS
+ArtWork,note,This column is a metacategory and only serves as field of communication between the project members with regards to this very entry,,2018-02-09,OS,2018-02-09,OS
+ArtWork,created,date of creation of line,,2018-02-09,OS,2018-02-09,OS
+ArtWork,created_by,acronym of project member creating this line,,2018-02-09,OS,2018-02-09,OS
+ArtWork,last_modified,date of last modification,,2018-02-09,OS,2018-02-09,OS
+ArtWork,last_modified_by,acronym of project member last modifying this line,,2018-02-09,OS,2018-02-09,OS
 ,,,,,,,
-Genre,genre_id,G and a two-digit number (leading zeros),http://id.loc.gov/authorities/genreForms.htmlv,2017-07-04,LH,2018-02-01,DP
+ArtForm,art_form_id,AF and a five-digit number (leading zeros),,2018-02-09,OS,2018-02-09,OS
+ArtForm,art_form_name,art form name in English; if necessary use Chinese pinyin. A guide for possible art forms is the following webpage: https://en.wikipedia.org/wiki/Fine_art,,2018-02-09,OS,2018-02-09,OS
```

Comment:
Since we are using MODS for everything else (and so does the Maoist legacy project) I d rather use their dictionary of ArtForms then wikipedias.

Reply:
MODS records objects. I am not sure whether this is helpful in our context, as we also have dances, oral stories and tales, songs, etc.  which are not recorded, but only remembered.